messagingpush/src/main/java/io/customer/messagingpush/util/PushTrackingUtil.kt

```diff
@@ -6,6 +6,7 @@ import io.customer.sdk.repository.TrackRepository
 
 interface PushTrackingUtil {
     fun parseLaunchedActivityForTracking(bundle: Bundle): Boolean
+    fun parseIntentExtrasForTrackingDelivered(bundle: Bundle): Boolean
```

Comment:
```suggestion
    fun trackMetricEvent(bundle: Bundle, event: MetricEvent): Boolean
```

Reply:
Resolving this as [discussed above](https://github.com/customerio/customerio-android/pull/204#discussion_r1193710660)